content/src/content/jcr_root/apps/core/wcm/components/teaser/v1/teaser/clientlibs/editor/js/teaser.js

```diff
@@ -198,4 +199,11 @@
             }
         }
     }
+
+    function associateDescriptionTextFieldWithLabel(descriptionTextfieldElement) {
+        var richTextContainerSelector = ".cq-RichText.richtext-container";
```

Comment:
Ideally the issue should be fixed in the RTE. We should only add an `arria-labelledby` as a fallback/workaround, if the RTE does not provide one.

Reply:
Addressed in c35eb464660b246a05ee2955a6f9c8b179a98d09.